src/PKCE.ts

```diff
@@ -71,6 +87,7 @@ export default class PKCE {
           Accept: 'application/json',
           'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8',
         },
+        ...this.corsRequestOptions
```

Comment:
Please include this new code in the `refreshAccessToken` method.

Reply:
As previously discussed [here](https://github.com/bpedroza/js-pkce/pull/36#issuecomment-1709358984), this is not required.